src/AlgebraicGeometry/ToricVarieties/CohomologyClasses/constructors.jl

```diff
@@ -5,8 +5,8 @@
 @attributes mutable struct CohomologyClass
     v::NormalToricVarietyType
     p::MPolyQuoRingElem
-    function CohomologyClass(v::NormalToricVarietyType, p::MPolyQuoRingElem)
-        @req parent(p) == cohomology_ring(v) "The polynomial must reside in the cohomology ring of the toric variety"
+    function CohomologyClass(v::NormalToricVarietyType, p::MPolyQuoRingElem; quick::Bool = false)
```

Comment:
```suggestion
    function CohomologyClass(v::NormalToricVarietyType, p::MPolyQuoRingElem, quick::Bool)
```
I'd prefer not to give this a default everywhere, especially not at such a core place. It can have a default for the user facing things, but this will make it easier to debug in the long run.